dashboard/app/controllers/lti/v1/account_linking_controller.rb

```diff
@@ -64,6 +66,19 @@ def new_account
           render status: :bad_request, json: {}
         end
       end
+
+      # POST /lti/v1/account_linking/unlink
+      def unlink
+        ao_id = params[:authentication_option_id]
+        return head :not_found if ao_id.blank?
+
+        ao = AuthenticationOption.find_by(id: params[:authentication_option_id])
```

Comment:
Super nit: you could use the variable `ao_id` here that's declared a few lines above instead of `params[:authentication_option_id]`